File: Back/src/ProEvents.Domain/Models/Speaker.cs
namespace ProEvents.Domain.Models
{
    public class Speaker{
        public int Id { get; set; }
        public string Name { get; set; }
        public string Bio { get; set; }
        public string ImgUrl { get; set; }
        public string Telephone {get; set;}
        public string Email { get; set; } 
        public IEnumerable<SocialNetwork> SocialNetwork { get; set; }
        public IEnumerable<SpeakerEvent> SpeakerEvents { get; set; }
    }
}

File: Front/ProEvents-App/src/app/models/Speaker.ts
import { SocialNetwork } from "./SocialNetwork";
import { SpeakerEvent } from "./SpeakerEvent";

export interface Speaker {
    id: number;
    name: string;
    bio: string;
    imgUrl: string;
    telephone: string;
    email: string;
    socialNetwork: SocialNetwork[];
    speakerEvents: SpeakerEvent[];
}
